tests/api/test_api.py

```diff
@@ -155,6 +155,40 @@ def test_is_node_schema_valid(cript_api: cript.API) -> None:
     assert cript_api._is_node_schema_valid(node_json=json.dumps(valid_file_dict), is_patch=False) is True
 
 
+def test_is_node_schema_valid_skipped(cript_api: cript.API) -> None:
+    """
+    test that a CRIPT node can be correctly validated and invalidated with the db schema, when skipping tests is active
+
+    * test db schema validation with an invalid node, and it should be invalid, but only detected if forced
+
+    Notes
+    -----
+    * does not test if serialization/deserialization works correctly,
+    just tests if the node schema can work correctly if serialization was correct
+
+    """
+
+    def extract_base_url(url):
+        # Split the URL by "//" first to separate the scheme (like http, https)
+        parts = url.split("//", 1)
+        scheme, rest = parts if len(parts) > 1 else ("", parts[0])
+
+        # Split the rest by the first "/" to separate the domain
+        domain = rest.split("/", 1)[0]
+        return f"{scheme}//{domain}" if scheme else domain
+
+    with cript.API(host=extract_base_url(cript_api.host), api_token=cript_api._api_token, storage_token=cript_api._storage_token) as local_cript_api:
+        local_cript_api.skip_validation = True
```

Comment:
will this not work with `os.getenv("CRIPT_HOST")` ?

this is a bit messy and if we can avoid it all the better, but if necessary then we can leave it in

Reply:
Using the environment variable here would make this brittle.
We are offering more ways then just ENV variables to specify the host, so relying on the fact that is in the env variables for this test is not a good idea.

The ideal solution is, that we offer a deepcopy implementation for API.

The second best solution is what I did here.
I can see that having this host separation function here isn't great.
But it is only in the tests.
we could also fix our `host` property of cript.API to actually only return the host address without the `api/v2` thing at the end.